src/mscorlib/mscorlib.shared.sources.props

```diff
@@ -777,6 +785,28 @@
     <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\FileStreamCompletionSource.Win32.cs" />
     <FileStreamSources Include="$(CoreFxSourcesRoot)\System\IO\Win32Marshal.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(FeatureCoreFxPath)' != 'true'">
+    <IoSources Include="$(BclSourcesRoot)\System\IO\Path.cs" />
```

Comment:
You can just delete these files. The TFS/git mirror was turned off and so it is no longer necessary to maintain the non-CoreCLR path in git.